central/image/datastore/store/v2/postgres/store.go

```diff
@@ -305,7 +305,7 @@ func copyFromImageComponentV2Cves(ctx context.Context, tx *postgres.Tx, iTime ti
 		"isfixable",
 		"fixedby",
 		"componentid",
-		"advisory",
+		"advisory_name",
```

Comment:
You need to update the input rows and the queries each time a CVE insert/update is called in this store, unfortunately.

Reply:
I have updated in both places